edenai_apis/apis/google/google_ocr_api.py

```
@@ -590,3 +591,52 @@ def ocr__financial_parser(
             original_response=Document.to_dict(document),
             standardized_response=standardized_response,
         )
+
+    async def ocr__afinancial_parser(
```

Pylint (4.0.3)

[refactor] Too many positional arguments (6/5)

(R0917)